packages/manager/modules/hub/src/components/payment-status-tile/AutorenewService.class.js

```diff
@@ -0,0 +1,81 @@
+import includes from 'lodash/includes';
+import snakeCase from 'lodash/snakeCase';
+
+import { DEBT_STATUS } from './payment-status-tile.constants';
+
+export default class AutorenewService {
+  constructor(service) {
+    Object.assign(this, service);
+  }
+
+  getRenew() {
```

Comment:
can you initialize renew variable in the constructor to avoid it calling a lot of if statements?

Reply:
This is fixed :)

This method is still present, since it has considerable amount of code. But we have a variable too